ultrasonic/src/main/kotlin/org/moire/ultrasonic/service/MediaPlayerService.kt

```diff
@@ -549,6 +564,14 @@ class MediaPlayerService : Service() {
         // Set Active state
         mediaSession!!.isActive = isActive
 
+        // TODO Implement Now Playing queue handling properly
+        mediaSession!!.setQueueTitle("Now Playing")
```

Comment:
I just wanted to try out how the Now Playing queue is displayed and handled in Android Auto, so this code is just a test, but works nicely.

Reply:
Now it is implemented correctly